docs/governance/CODE_OF_CONDUCT.md

```diff
@@ -14,7 +14,7 @@ We pledge to act and interact in ways that contribute to an **open**, **welcomin
 
 ## Our Standards
 
-Examples of **required** behavior that contributes to a positive environment for our
+Examples of **standard** behavior that contributes to a positive environment for our
```

Comment:
"required" is more clear to us. we equate "standard" to "common" and don't always follow what's common.

Reply:
I agree, it felt odd when I read it in the moment that's why I replaced it but I can see it making more sense.